images/win/toolset-2016.json

```diff
@@ -47,6 +47,26 @@
                 "12.*",
                 "14.*"
             ]
+        },
+        {
+            "name": "Boost",
+            "url" : "https://raw.githubusercontent.com/akv-platform/boost-hostedtoolcache/master/versions-manifest.json",
+            "arch": "x86_64",
+            "platform" : "win32",
+            "toolset": "msvc14.1",
+            "versions": [
+                "1.69.0"
+            ]
+        },
+        {
+            "name": "Boost",
+            "url" : "https://raw.githubusercontent.com/akv-platform/boost-hostedtoolcache/master/versions-manifest.json",
+            "arch": "x86_64",
+            "platform" : "win32",
+            "toolset": "msvc14.2",
+            "versions": [
+                "1.71.0"
+            ]

```

Comment:
I think both versions on Win2016 should be with msvc14.1 toolset. See toolcache-2016.json